src/commands/protocols/common/network-protocol-command.js

```diff
@@ -17,11 +17,12 @@ class NetworkProtocolCommand extends Command {
      * @param command
      */
     async execute(command) {
-        const { blockchain, operationId, minimumNumberOfNodeReplications } = command.data;
+        const { blockchain, operationId, minimumNumberOfNodeReplications, batchSize } =
+            command.data;
 
         this.operationIdService.emitChangeEvent(this.operationStartEvent, operationId, blockchain);
 
-        const batchSize = this.operationService.getBatchSize();
+        const batchSizePar = this.operationService.getBatchSize(batchSize);
         const minAckResponses = this.operationService.getMinAckResponses(
```

Comment:
no need to use function here we can just do `minAckResponses = minimumNumberOfNodeReplications  ?? CONSTANT`